go/tools/go_bin_runner/main.go

```diff
@@ -20,6 +20,9 @@ var GoBinRlocationPath = "not set"
 var ConfigRlocationPath = "not set"
 var HasBazelModTidy = "not set"
 
+var bazelWorkingDir = os.Getenv("BUILD_WORKING_DIRECTORY")
+var bazelWorkspaceDir = os.Getenv("BUILD_WORKSPACE_DIRECTORY")
```

Comment:
There's an `os.Getenv("BAZEL")` call below. Should we maybe extract this to a function loadEnv() that returns a structure and is called once?